### DISCUSSION TITLE
FN6 Feedback

### DISCUSSION THREAD

**Discussion by tdegeorge** [TeamBasedInquiryLearning/precalculus#107] [Show and tell]
Writer: @kathypinzon 
Reviewer: @tdegeorge

**Comment by tdegeorge** [comment]
I have no idea if I started this right or not, but here we go! :)

**Reply by siwelwerd** [reply]
This is perfect, thanks Tonya!

**Comment by tdegeorge** [comment]
Activity 2.6.2: I think "temperate" is supposed to be "temperature" in part c.
@kathypinzon

**Comment by tdegeorge** [comment]
Activity 2.6.2: Idea for part d. So, maybe end the activity with part c and then go into a remark, like "notice that when you converted 68 degrees Fahrenheit, you got a value of 20 degrees Celsius. Alternatively, when you converted 20 degrees Celsius, you got 68 degrees Fahrenheit. This indicates that the equation you were given for C and the equation you found for F are inverses."  or something like that. 

Then, in another activity (after the definition of inverses), you can bring back the idea of composition and have them prove they are inverses?
@kathypinzon

**Comment by tdegeorge** [comment]
Activity 2.6.11: The wording for part c is a little confusing. I think it's because of the extra comma?

**Comment by tdegeorge** [comment]
Activity 2.6.13: I would switch part a and b since part b is more similar to what they just did and part a is harder.

Is part c supposed to have MC options?

Should we add a remark about needing to factor (like part c)?

**Comment by tdegeorge** [comment]
It's mentioned that another way to find an inverse is to switch the x and the y. Should we have another activity that specifically does that?

**Reply by kathypinzon** [reply]
took, part (c) from 2.6.13 and used it to demonstrate this method.  Then used 2.6.13 part (a) and (b) as example at the end - changing directions to say "use either method"
